### DISCUSSION TITLE
V-app-bar scroll-behavior

### DISCUSSION THREAD

**Discussion by random362** [vuetifyjs/vuetify#18830] [Q&A]
Hi everyone.
I'm trying to make a two-part header: regular + sticky on scroll.
According the [doc](https://vuetifyjs.com/en/components/app-bars/#scroll-behavior)
> **hide:** The default slot area will shift up and hide as the user scrolls down. _The extension slot remains visible_.

But cannot achieve the expected behavior - the extension slot shift up with default one.

[Sandbox](https://play.vuetifyjs.com/#eNqVUU1LxDAU/CuP3ATbdFm9lFjw4M2jN+sh275lg2kSkrQopf/d1+6m+4EIXpI3k2FmeHkfWfANf3YuH3pkJRMRO6dlxKo2AGLIpHPLmEC2kx7ohtB4q3W2w4MclPVPNTuoFmt2UpM+WcGQBW1j2eJe9jqugkvLLKqosXqbT8Fv6dWSX9X7NQW/IpqgrPkr5yWJ4L+JF9LzXjqpznGEG2siUZgkC+2o4X5ZlAJlYFMUNYPyE79nhvY2jqBgmu7g1XrsQLnQd9Babb3g6Q+O+bf2M7dWOBWk+ao7Qfox5SIEjL2rBD9CemLTPdvm23xTsHl4yB/Zxw+z3qoT)

Can anyone help me?

**Comment by nickmadev** [comment]
@random362 
Your questions is opening issue in this project.
https://github.com/vuetifyjs/vuetify/issues/18020
If you want alternative way, you can implement it by css styling.
